install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -220,7 +220,9 @@ spec:
                 done
               else
                 # COS-beta (with containerd)
-                for f in `find /var/lib/cni/networks/ -type f ! -name lock ! -name last_reserved_ip.0`; do crictl stopp $(cat $f) || true; done
+                PATH=${PATH}:{{ .Values.cni.binPath }}
```

Comment:
I would call `crictl` by its absolute path instead of manipulating `PATH`. Why is `crictl` in `cni.binPath`? :thinking: Has this changed in recent versions of COS?

Reply:
I am not sure if that changed in previous version of COS. But I agree that it would be better to explicitely call `crictl` through its absolute path: `/home/kubernetes/bin/crictl`. I'll change that.